pkgs/development/python-modules/remi/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv
+, lib
+, buildPythonPackage
+, fetchPypi
+, setuptools-scm
+}:
+
+buildPythonPackage rec {
+  pname = "remi";
+  version = "2022.7.27";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-VC34jhdLvVj+djxJ5cIRbyyEE8GpNHMQae2MF7uc1kw=";
+  };
+
+  nativeBuildInputs = [
+    setuptools-scm
+  ];
+
+  # We're unable to deselect "optional" integrations tests
+  # with unpackaged libraries (pyepics)
+  doCheck = false;
```

Comment:
With `unittest` tests, it's possible to "disable" individual tests by using e.g. `substituteInPlace` to make them either not start with `test_` or `Test` (I tend to turn them into e.g. `dont_test_`), and `unittest` won't pick them up.

Reply:
How hard would it be to package pyepics?

Reply:
Packaging pyepics is far from easy. There are many forks and dead dependencies. For a simple widget test it is not worth it.
I managed to deselect the test using pytest and lots of patching.